sljit_src/sljitExecAllocator.c

```diff
@@ -192,6 +192,63 @@ static SLJIT_INLINE void free_chunk(void *chunk, sljit_uw size)
 
 #endif
 
+#if defined(__APPLE__) && defined(__arm64__)
```

Comment:
this is also needed in x86, to test will need to have the test application linked against the Hardened Runtime though.

once the '__arm64__' define is removed it might make more sense to get all this code together with the rest of the __APPLE__ specific code in line 97